gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 155
```

Comment:
How come you need to bump this, and is there a corresponding runtime-os PR?

Reply:
Yes, I wanted to test this change with `corda-runtime-os` and thought the easiest would be to bump the version. PR: https://github.com/corda/corda-runtime-os/pull/1853